tests/test_server.py

```
@@ -1776,6 +1848,317 @@ def test_get_ticket_resource(ticket_id: str) -> str:
+        )
+
+        server_instance.client.get_ticket.return_value = ticket.model_dump()  # type: ignore[union-attr]
+
+        # Setup the resource and capture it
+        server_instance._setup_resources()
+        test_resources = {}
+
+        original_resource = server_instance.mcp.resource
+
+        def capture_resource(uri_template: str) -> Callable[[Callable[..., Any]], Any]:
+            def decorator(func: Callable[..., Any]) -> Any:
```

Ruff (0.14.1)

Dynamically typed expressions (typing.Any) are disallowed in `decorator`

(ANN401)